vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Microsoft/vcpkg README
+description: README for vcpkg registry
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: shell-selections
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 10/17/2023
+ms.prod: vcpkg
+ROBOTS: NOINDEX
+---
+
+# vcpkg: Overview
+
+vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.
```

Comment:
I suggest adding the opening paragraph from the overview page documentation here instead of this sentence (for consistency): 

vcpkg is a free and open-source C/C++ package manager maintained by Microsoft and the C++ community. Initially launched in 2016 as a tool for assisting developers in migrating their projects to newer versions of Visual Studio, vcpkg has evolved into a cross-platform tool used by developers on Windows, macOS, and Linux. vcpkg has a large catalog of open-source libraries and enterprise-ready features designed to facilitate your development process with support for any build and project systems. vcpkg is a C++ tool at heart and is written in C++ with scripts in CMake. It is designed from the ground up to address the unique pain points C/C++ developers experience.

Reply:
Also worth adding a link to our docs at the end of the paragraph.